### ISSUE TITLE
Render multi-root workspace folders as an accordion, rather than folders

### ISSUE THREAD

**Issue by Ratismal** [microsoft/vscode#74229]
Would it be possible to have a configuration setting to display workspace folders as accordion tabs, rather than folders? I'm having an issue where it's difficult for me to tell exactly which workspace I'm currently in, and have trouble differentiating workspace folders and actual files at a glance. This is especially troublesome when there are a lot of files and subfolders in each workspace folder.

![O08YKhwnnp](https://user-images.githubusercontent.com/1059362/58283875-b0ed7900-7d66-11e9-8900-8a44d597ef97.gif)

This is a (non-functional) mockup of what I mean by "accordion tabs": 
![image](https://user-images.githubusercontent.com/1059362/58284636-42a9b600-7d68-11e9-9c78-d14f161a4d24.png)

Each workspace folder has it's own section in the split view and can be expanded/collapsed individually, while still displaying at all times. Having this option would make traversing my workspace a lot easier :3 

If this option could be implemented, it would be fantastic! If you believe this to be better suited as an extension instead, could I at least ask to be pointed in the right direction for interfacing with the split view?

Thank you!

**Comment by isidorn** [comment]
This is a fair feature request and I will leave it open (even though we do not currently have it planned).
An extension could contribute a brand new view in the viewlet , which could in theory represent things like this.
However it is not possible for the extension to customise the current explorer view in this way

**Comment by nicolo-ribaudo** [comment]
I don't think that this should be limited to workspaces with different root folders. When I'm working on a monorepo with a lot of different packages (e.g. https://github.com/babel/babel/tree/master/packages), working accross different packages "alphabetically distant" leads to a bad DX.
The file explorer should also allow duplicating the same file tree, like split editor view allows viewing different positions of the same file.

**Comment by 1138-4EB** [comment]
@isidorn, can we please have this issue tagged with [`workbench-multiroot`](https://github.com/microsoft/vscode/issues?utf8=%E2%9C%93&q=label%3Aworkbench-multiroot+)?

---

> [@nicolo-ribaudo] The file explorer should also allow duplicating the same file tree, like split editor view allows viewing different positions of the same file.

A similar feature was suggested in [#76891](https://github.com/microsoft/vscode/issues/76891#issuecomment-509288763).

---

I think that this issue might be related to #76399. Here, a separate TreeView for each multi-root workspace folder is requested. In #76399, a separate Editor Group for each multi-root workspace folder is requested.

**Comment by isidorn** [comment]
@1138-4EB sure. I have added that label.

**Comment by eine** [comment]
#76904 is now a candidate for `backlog`. Since this issue is explicitly included in that proposal, I'd like to please ask users interested on this feature to consider upvoting #76904.

**Comment by LucaGabi** [comment]
please also read this feature request here: #95182 in my opinion it is similar request with less clutter
